### PR TITLE
[Board]: Support for M5Stack CoreS3

### PR DESCRIPTION
**Pull request by lboue**
Add support for M5Stack CoreS3 variant:

- [M5Stack CoreS3](https://shop.m5stack.com/products/m5stack-cores3-esp32s3-lotdevelopment-kit)
- [LoRa868_V1.1 Module](https://shop.m5stack.com/products/lora-module-868mhz-v1-1) with RA01H (SX1276)

- [x] Radio LoRa part
- [ ] LCD display

image

### CODE REVIEW

File: src/mesh/generated/meshtastic/mesh.pb.h [hunk 1]
```diff
@@ -109,7 +109,7 @@ typedef enum _meshtastic_HardwareModel {
     meshtastic_HardwareModel_NRF52840_PCA10059 = 40,
```
Comment (fifieldt) [inline src/mesh/generated/meshtastic/mesh.pb.h:109]: These files are automatically generated. You probably want to propose a change to 
https://github.com/meshtastic/protobufs

**Review by fifieldt — DISMISSED** [review]

**Comment by fifieldt** [comment]
Thanks for your first patch! Welcome to the community :)

File: src/mesh/generated/meshtastic/mesh.pb.h (hunk 1 above)
Comment (lboue) [reply src/mesh/generated/meshtastic/mesh.pb.h:109]: Thanks, I'll do that.

**Comment by lboue** [comment]
Done: [Add support for M5Stack CoreS3 variant #585](https://github.com/meshtastic/protobufs/pull/585)

**Comment by prusnak** [comment]
> * [LoRa868_V1.1 Module](https://shop.m5stack.com/products/lora-module-868mhz-v1-1) with RA01H (SX1276)

Are you going to work on adding support for `LoRa868_V1.1`? So far this PR only adds CoreS3 support

**Comment by lboue** [comment]
I manged to send a LoRa message to a Heltech node from my CoreS3.

**Comment by lboue** [comment]
Closing this in favor of #5049

**Pull request closed without merging**